contentful_go.go

```diff
@@ -15,29 +14,29 @@ const baseUrl string = "https://cdn.contentful.com/"
 
 type Contentful struct {
 	spaceId, accessToken string
+	client               *http.Client
 }
 
 // Creates contentful client
 // Example:
-//  		client := contentful.CreateClient('SPACE_ID', 'DELIVERY_ACESS_TOKEN')
+//  		client := contentful.New('SPACE_ID', 'DELIVERY_ACESS_TOKEN')
 //
-func CreateClient(spaceId, accessToken string) Contentful {
-	return Contentful{spaceId, accessToken}
+func New(spaceId, accessToken string) Contentful {
+	return Contentful{spaceId, accessToken, &http.Client{}}
 }
 
 // Gets an Entry
 // Example:
-//  		client := contentful.CreateClient("SPACE_ID", "ACCESS_TOKEN")
+//  		client := contentful.New("SPACE_ID", "ACCESS_TOKEN")
 //  		entry, _ := client.GetEntry("ENTRY_ID")
 //  		fmt.Printf("got entry with id %s", entry.Sys.Id)
 // It returns a pointer to an Entry object prefilled with response data
 func (c Contentful) GetEntry(entryId string, query map[string]string) (entry Entry, err error) {
 	e := Entry{}
-	reader, err := c.performeRequest("GET", "spaces/"+c.spaceId+"/entries/"+entryId, query)
+	body, err := c.performRequest("GET", "spaces/"+c.spaceId+"/entries/"+entryId, query)
```

Comment:
Not a comment on this commit, but to give an example of what I was talking about earlier maybe you would have something like this near the start of this file:

```
const entryRequestURLFmt = "spaces/%s/entries/%s"
```

Then in this function:

```
body, err := c.performRequest("GET", fmt.Sprintf(entryRequestURLFmt, c.spaceId, entryId), query)
```

And maybe even assemble the URL on a previous line to make it a bit more obvious (and later maybe log it for debugging etc).